test/module/irohad/multi_sig_transactions/mst_processor_test.cpp

```diff
@@ -48,6 +49,7 @@ class MstProcessorTest : public testing::Test {
   /// use effective implementation of storage
   std::shared_ptr<MstStorage> storage;
   std::shared_ptr<FairMstProcessor> mst_processor;
+  const shared_model::crypto::Keypair kMyKeypair = makeKey();
```

Comment:
No so reliable to create effective dependencies for components in unit-test better to rework it with mock.

Reply:
I have done it the same way it is done in this file, using the same function and types. If it is wrong, then the entire thing should be changed in a separate patch.